### DISCUSSION TITLE
Theme Generator

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#1054] [Q&A] [answered]
Seems that some of the text (tried to highlight where theme matches and where it doesn't) is inconsistent with the text color. Is this purposeful? 

![image](https://user-images.githubusercontent.com/78372056/136455271-5e5cc753-5490-49db-bc68-e9615c6b6475.png)

**Comment by mturoci** [comment, marked as answer]
Hi @AlexWennerDT, yes that's on purpose. For example, you wouldn't want to have links with same color as default text. You want to have them distinguished (table example in your screenshot). Same goes for string literals within `Copy Code` although they are not really part of Wave.

**Reply by Alex-Wenner-FHR** [reply]
Understandable, my only challenge in that case would be the header title/subtitle... IMO it would be more ideal to have that match the text color/customizable by itself. Purely feedback! This is an awesome feature nonetheless. Thanks!

**Reply by mturoci** [reply]
Good point. It will need more thought though since we try to support also 2-color themes (primary and text are same - default black white theme).

**Reply by Alex-Wenner-FHR** [reply]
Understood! Also maybe the two table columns having different colors (Name/Surname respectively). Going to mark this as answered, thanks @mturoci !

**Reply by mturoci** [reply]
>Also maybe the two table columns having different colors (Name/Surname respectively)

I could have ordered columns better, that's true :)
